data/core/help.cfg

```diff
@@ -285,18 +285,15 @@ Normally you can undo a unit’s movement, as long as an event with a randomized
     [topic]
         id=saveload
         title= _ "Save-loading"
-        text= _ "Random numbers are part of Wesnoth, attacks can fail and units can die due to bad luck. This is an expected part of the game, and one of the aims of the game is that a sufficiently skilled player should be able to complete all of the mainline campaigns, on hard, without needing to save-load. Going back to a previous turn to try a different strategy is a part of learning the game, but we recommend against reloading merely to try the same strategy again while hoping for better luck." + _ "
+        text= _ "Random numbers are part of Wesnoth, attacks can fail and units can die due to bad luck — this is an expected part of the game. Going back to a previous turn to try a different strategy is a part of learning the game, but we recommend against reloading merely to try the same strategy again while hoping for better luck." + "
 
-One of the challenges of the game is to work out how to protect your heroes. Small risks quickly build up: if you have five important units, and they each have just a 1% chance of death each turn, you can <italic>text='expect'</italic> one of them to die about every 20 turns. So, you’ll rarely make it through a scenario without having it happen." + _ "
+" + _ "One of the challenges of the game is to work out how to protect your heroes. Small risks quickly build up: if you have five important units, and they each have just a 1% chance of death each turn, you can <italic>text='expect'</italic> one of them to die about every 20 turns. So, you’ll rarely make it through a scenario without having it happen." + "
 
-The following points are under discussion on the forums. At least for the 1.15.x series before 1.16.0 please assume that the forum threads are more authoritative than the text on this page." + _ "
+" + _ "Small risks build up for your non-hero units too — however these units don’t have to survive. While the story and dialogue usually aim to give you some emotional bond with your troops, the game mechanics reward cold-blooded thoughts such as “even if those units survive, I’m not going to recall them in any future scenarios, ...”, and the even colder “..., and that will reduce my upkeep”." + "
```

Comment:
I'd change:

> the game mechanics reward cold-blooded thoughts such as “even if those units survive, I’m not going to recall them in any future scenarios, ...”, and the even colder “..., and that will reduce my upkeep”

to:

"the game mechanics also often reward purely practical actions, such as making sure the first units to die are ones you have no intention of recalling in future scenarios."

Reply:
I've pushed a new draft that goes in a different direction: "the game mechanics reward remembering that it’s just a game — there is rarely any penalty for letting these units die, other than being unable to recall them in future scenarios. Your battle-plan should distribute the risks based on which troops you want to recall later."

Reply:
"battle-plan" shouldn't have a dash in it, but otherwise I'm good with that.

Reply:
Resolved in the version pushed to master.